Classes/EventStore/RawEvent.php

```diff
@@ -46,23 +46,29 @@
     private $version;
 
     /**
+     * @var int
+     */
+    private $sequenceNumber;
+
+    /**
      * @var \DateTimeInterface
      */
     protected $recordedAt;
 
-    public function __construct(string $identifier, string $type, array $payload, array $metadata, int $version, \DateTimeInterface $recordedAt)
+    public function __construct(int $sequenceNumber, string $type, array $payload, array $metadata, int $version, string $identifier, \DateTimeInterface $recordedAt)
     {
```

Comment:
I wasn't sure about the argument order, but it doesn't really matter as that's not api/userland